.coafile

```diff
@@ -40,7 +40,7 @@ ignore_length_regex = ^.*https?://
 [DOCS]
 bears = SpaceConsistencyBear, LineLengthBear
 
-files = docs/**/*.rst, CONTRIBUTING.rst, README.rst
+files = **.rst
```

Comment:
On command prompt I re-checked.
Python 3.4.2 (v3.4.2:ab2c023a9432, Oct  6 2014, 22:15:05) [MSC v.1600 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from coalib.parsing.Globbing import glob
>>> glob("**.rst")
['CONTRIBUTING.rst', 'docs\\Developers\\Adding_CI.rst', 'docs\\Developers\\Bear_Installation_Tool.rst', 'docs\\Developers\\coala_settings.rst', 'docs\\Developers\\Codestyle.rst', 'docs\\Developers\\Development_Setup.rst', 'docs\\Developers\\Executing_Tests.rst', 'docs\\Developers\\External_Bears.rst', 'docs\\Developers\\Git_Basics.rst', 'docs\\Developers\\Linter_Bears_Advanced.rst', 'docs\\Developers\\Newcomers_Guide.rst', 'docs\\Developers\\Review.rst', 'docs\\Developers\\Testing_Bears.rst', 'docs\\Developers\\Useful_Links.rst', 'docs\\Developers\\Writing_Documentation.rst', 'docs\\Developers\\Writing_Good_Commits.rst', 'docs\\Developers\\Writing_Linter_Bears.rst', 'docs\\Developers\\Writing_Native_Bears.rst', 'docs\\Developers\\Writing_Tests.rst', 'docs\\index.rst', 'README.rst', 'RELEASE_NOTES.rst']